qa/workunits/rbd/rbd_mirror.sh

```diff
@@ -68,6 +68,16 @@ POOL=mirror
 SRC_DIR=$(readlink -f $(dirname $0)/../../../src)
 TEMPDIR=
 
+# These vars facilitate running this script in an environment with
+# ceph installed from packages, like teuthology. These are not defined
+# by default.
+#
+# USE_EXISTING_CLUSTER - if set, do not start and stop ceph clusters
+# UNMANAGED_RBD_MIRROR - if set, do not start and stop rbd-mirror
+# CEPH_ID              - id portion of client.$id ceph user that rbd-mirror uses
+#                        this is assumed to be the same for all clusters, and is
+#                        only used if UNMANAGED_RBD_MIRROR is set
+
```

Comment:
I would prefer such vars follow the convention to have RBD_MIRROR_ prefix. Or do they are used by teuthology not only for rbd_mirror? Also, if CEPH_ID is used only together with  UNMANAGED_RBD_MIRROR, may be we can combain them? I.e. what about these?
```
RBD_MIRROR_USE_EXISTING_CLUSTER = empty|non-empty
RBD_MIRROR_USE_RBD_MIRROR = empty|${CEPH_ID}
```